draft-ietf-homenet-front-end-naming-delegation.mkd

```diff
@@ -649,25 +615,27 @@ The HNA SHOULD reject any incoming messages other than DNS NOTIFY response, SOA
 {{!RFC7368}} in Section 3.7.3 recommends DNSSEC to be deployed on both the authoritative server and the resolver.
 The resolver side is out of scope of this document, and only the authoritative part of the server is considered.
 
-This document assumes the HNA signs the Public Homenet Zone.
+It is RECOMMENDED the HNA signs the Public Homenet Zone.
 
 Secure delegation is achieved only if the DS RRset is properly set in the parent zone.
 Secure delegation is performed by the HNA or the DOIs.
```

Comment:
The secure delegation requires the DS RRSet to be added in the parent zone. The decision thus rather depends on who the parent zone is likely to trust. In most cases, I do not expect the homenet to be allowed to update the parent zone. It is expected though that the DOI is also a registrar in which case, it will have a specific relation with the parent zone. 
I slightly changed the text and combined the two paragraphs:

Secure delegation is achieved only if the DS RRset is properly set in the parent zone.
Secure delegation can be performed by the HNA or the DOIs and the choice highly depends on which entity is authorized to perform such updates.
Typically, the DS RRset can be updated manually in the parent zone with nsupdate for example.
This requires the HNA or the DOI to be authenticated by the DNS server hosting the parent of the Public Homenet Zone.
Such a trust channel between the HNA and the parent DNS server may be hard to maintain with HNAs, and thus may be easier to establish with the DOI.
In fact, the Public Authoritative Server(s) may use Automating DNSSEC Delegation Trust Maintenance  {{!RFC7344}}.